extensions/rocksdb-repos/RocksDbRepository.cpp

```diff
@@ -20,22 +20,30 @@ using namespace std::literals::chrono_literals;
 
 namespace org::apache::nifi::minifi::core::repository {
 
-void RocksDbRepository::printStats() {
+std::optional<RepositoryMetricsSource::RocksDbStats> RocksDbRepository::getRocksDbStats() const {
```

Comment:
This function is identical to `DatabaseContentRepository::getRocksDbStats()`.  Could we move the common code somewhere and call it from these two places, to avoid duplication?

Reply:
Updated in 0e772dd8719c74576dd9cf39be7fe395965afc96